src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+public class Storage {
+
+    protected String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads task list from file
+     *
+     * @return List of task in file
+     * @throws IOException
+     */
+    public ArrayList<Task> load() throws IOException {
+        ArrayList<Task> tasks = new ArrayList<Task>(100);
+        try {
+            File data = new File(filePath);
+            Scanner dataReader = new Scanner(data);
+            while (dataReader.hasNextLine()) {
+                String task = dataReader.nextLine();
+                System.out.println(task);
+                String[] split = task.split(" , ");
+                String type = split[0];
+                boolean done;
```

Comment:
Maybe naming this boolean as isDone or hasCompletedTask might be better? (since done may be read as a verb)